core-web/libs/portlets/dot-ema/src/lib/services/dot-page-api.service.ts

```diff
@@ -18,6 +24,9 @@ export interface DotPageApiResponse {
         language: DotLanguage;
         persona?: DotPersona;
     };
+    layout: DotLayout;
+    template: DotTemplate;
+    containers: DotPageContainerStructure;
```

Comment:
Why don't we do a `Pick` here of what we need? is just an idea so we can easily mock stuff around, well if that helps.